packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -1913,3 +1913,18 @@ function VM-Unzip-Recursively {
         }
     }
 }
+# Obtains the chocolatey install script path from a package, for example: C:\ProgramData\chocolatey\lib\capa.vm\tools\chocolateyinstall.ps1
+# Reads the tag with name tags from the nuspec file,
+# Returns the category if the tags exist in the nuspec
+function VM-Get-Category {
+    $installPath = $MyInvocation.MyCommand.Definition
+    $toolDir = "$(Split-Path -parent (Split-Path -parent $installPath))"
+    $packageName = Split-Path -Path (Split-Path -parent $installPath) | Split-Path -Leaf
+    $nuspec = $packageName + ".nuspec"
+    if (Test-Path ($toolDir)){
```

Comment:
the directory must exist, otherwise we return an invalid category here

Reply:
@Ana06 if $toolDir doesn't exist, the return value should be "" for example?

Reply:
ah ok, so I remove the condition Test-Path